packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/03-quickstart.mdx

```diff
@@ -103,5 +109,4 @@ The goal of this library is to make it easy to build any type of video/calling e
 - Use our library of built-in components.
 - Mix & Match between your own and built-in components.
 
-You can use and customize the [built-in components](../../ui-components/overview).
-If you decide to build your own components we have a dedicated [UI Cookbook section](../../ui-cookbook/overview) to help you get started.
+The built-in components you can customize using theming and props. If you decide to build your own components we have a dedicated [UI Cookbook section](../../ui-cookbook/overview) to help you get started.
```

Comment:
If you're using our built-in components, you can easily customize them through theming and props. For creating your own components, the UI Cookbook section is there to help you get started.